src/phel/json.phel

```diff
@@ -0,0 +1,48 @@
+(ns phel\json
+  (:use \Exception))
+
+(defn valid-key? [v]
```

Comment:
Can we make this private?
```suggestion
(defn- valid-key? [v]
```

Reply:
See my comment on this: https://github.com/phel-lang/phel-lang/pull/489#issuecomment-1259005195

Reply:
I see. This `valid-key?` is used only in the `encode-value`, I don't see why we want to keep it public? Or is there any other reason?